roles/master/tasks/main.yml

```diff
@@ -10,6 +10,17 @@
   tags:
     - master
 
+- name: install kubernetes node
```

Comment:
to reduce the number of package installation sections, leverage with_items on the previous task